UI/xdg-data/com.obsproject.Studio.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.obsproject.Studio</id>
+  <launchable type="desktop-id">com.obsproject.Studio.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>OBS Studio</name>
+  <developer_name>OBS Project</developer_name>
+  <summary>Live streaming and video recording software</summary>
+  <description>
+    <p>Free and open source software for video recording and live streaming.</p>
+  </description>
+  <url type="homepage">https://obsproject.com</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://obsproject.com/assets/images/OBSDemoApp.jpg</image>
+    </screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1"/>
+  <releases>
+    <release version="23.0.2" date="2019-03-08"/>
```

Comment:
This should be auto-generated.

Reply:
But please keep the build reproducible!

Reply:
I mean something like this https://github.com/obsproject/obs-studio/pull/1629/files#diff-af3b638bc2a3e6c650974192a53c7291R42

It does a string replace to generate a final file.

Reply:
I know. You could replace the version string, but the date can't be a build timestamp.

Multiple release entries are also possible:
```
<releases>
  <release version="23.0.2" date="2019-03-08"/>
  <release version="23.0.1" date="2019-XX-XX"/>
  <release version="23.0.0" date="2019-XX-XX"/>
  ...
</releases>
```

Reply:
You can use CMake to generate the current date of the CMake run.
```
string(TIMESTAMP TODAY "%Y-%m-%d")
```
Generating multiple release elements could be done by either:

1. We maintain a public list of release dates/elements to be fetched on build
2. The build process iterates through all git tags and uses the tag date as the "release date"

Reply:
Using the current date would make the build non-reproducible.

Reply:
You could still use the git tag date, which would be immutable.

Reply:
That's what I'd do - the git tag with its date, unless the user is building a non-tagged commit, then build with today's date.